src/paper/13-q-and-a.ts

```diff
@@ -22,7 +22,7 @@ In the case of a long indoor hangout, we suggest just to cap the total Activity
 
 ### If you live with multiple people, do you account for the additional risk of a longer infection chain?
```

Comment:
Maybe add "within the household" at the end of this question to make it extra clear